apps/fabric-website-resources/src/FocusZone.test.dom.tsx

```diff
@@ -0,0 +1,32 @@
+import * as puppeteer from 'puppeteer';
+
+describe('FocusZone', () => {
+  it('can use arrows vertically', async () => {
+    // Arrange
+    const browser = await puppeteer.launch({ headless: true });
+    const page = await browser.newPage();
+    await page.goto('http://localhost:4322/#examples/focuszone');
```

Comment:
Is there a chance we could ever end up with multiple processes running tests on the same machine at the same time? In that case the port would be different for the second process.

Reply:
I didn't think we allow for that now... There's no coordination of these ports either. So I think this is fine...

Reply:
Actually we do this to coordinate the ports:
https://github.com/OfficeDev/office-ui-fabric-react/blob/master/scripts/tasks/webpack.js#L10-L13

Reply:
My next step with this is connect with a couple of devs to write actual tests. Depending on the outcome, I might remove webpack completely. So, I'd like to leave that for later consideration.